grails-app/domain/org/pih/warehouse/inventory/CycleCountSummary.groovy

```diff
@@ -31,7 +32,7 @@ class CycleCountSummary implements Serializable {
     String blindCountVarianceComment
 
     // Verification count details
-    Date verificationCountDateCounted
+    Instant verificationCountDateCounted
```

Comment:
see cycleCounDetails view. This view is based on that one and so have the same field sources